app/install/source.md

```diff
@@ -11,90 +11,144 @@ redirect_from: /install/compile/
 {% capture luajit_version %}{{site.data.kong_latest.dependencies.luajit}}{% endcapture %}
 {% capture luarocks_version %}{{site.data.kong_latest.dependencies.luarocks}}{% endcapture %}
 {% capture openresty_version %}{{site.data.kong_latest.dependencies.openresty}}{% endcapture %}
+{% capture openssl_version %}{{site.data.kong_latest.dependencies.openssl}}{% endcapture %}
+{% capture libyaml_version %}{{site.data.kong_latest.dependencies.libyaml}}{% endcapture %}
+{% capture pcre_version %}{{site.data.kong_latest.dependencies.pcre}}{% endcapture %}
+{% capture gopluginserver_version %}{{site.data.kong_latest.dependencies.gopluginserver}}{% endcapture %}
```

Comment:
does this also need to be removed given we are not including go-pluginserver as a dep?